neurom/view/view.py

```diff
@@ -574,7 +574,7 @@ def dendrogram(obj, show_diameters=True, new_fig=True, new_axes=True, subplot=Fa
     kwargs['xlabel'] = kwargs.get('xlabel', 'micrometers (um)')
     kwargs['ylabel'] = kwargs.get('ylabel', 'micrometers (um)')
     kwargs['no_legend'] = False
-    kwargs['aspect_ratio'] = "auto"
+    kwargs['aspect_ratio'] = kwargs.get('aspect_ratio', 'auto')
```

Comment:
This ensures that if the user selected a different value it will overwrite the default.